dev/CalendarView/APITests/CalendarViewTests.cs

```diff
@@ -23,7 +23,7 @@ namespace Windows.UI.Xaml.Tests.MUXControls.ApiTests
     [TestClass]
     public class CalendarViewTests : ApiTestBase
     {
-        [TestMethod]
+        // TODO: fix failing tests after color updates [TestMethod]
```

Comment:
Please file an issue on this.

Reply:
In addition to that, we should use `[TestProperty("Ignore", "True")]` instead of commenting out the testmethod attribute.

Reply:
#3906

Reply:
Yes, as Marcel suggested, please use [TestProperty("Ignore", "True")] to disable tests, it allows the test to show up in the test explorer

---
In reply to: [553578719](https://github.com/microsoft/microsoft-ui-xaml/pull/3887#discussion_r553578719) [](ancestors = 553578719)